packages/subgraph/subgraph.yaml.js

```diff
@@ -1,18 +1,32 @@
+const fs = require("fs");
+
+const { addresses } = require("@liquity/lib-ethers/deployments/mainnet.json");
+// const { addresses } = require("@liquity/lib-ethers/deployments/dev.json");
+
+// https://etherscan.io/tx/0x0b612c6ffcef059a1cb9ceda83dee44a3d74e35d5018f1b8b486f3186cd7850e
+const startBlock = 12178551;
+// const startBlock = 0;
+
+const yaml = (strings, ...keys) =>
```

Comment:
Why do we need this and the tagged templates? Wouldn’t just work if we simply define:
```
const manifest = `
specVersion: 0.0.2
(...)
      address: "${addresses.troveManager}"
      startBlock: ${startBlock}
      (...)
`
```
The variables would already be replaced, right?

Reply:
I think the whole thing would be simpler as just a tagged template literal

Reply:
Yeah, I didn't want to lose syntax highlighting.

Reply:
Oh, I see! Cool, thanks for the clarification!